app/metrics.yaml

```
@@ -254,3 +260,249 @@ control:
       - fxr-telemetry@mozilla.com
       - dmu@mozilla.com
     expires: "2020-11-01"
+
+pages:
+  page_load:
+    type: timing_distribution
+    description: >
+      Measuring how long each page takes to load.
+    time_unit: nanosecond
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2279
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/xxx
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-11-01"
+
+immersive:
+  duration:
+    type: timing_distribution
+    description: >
+      Measuring how long users stay in the immersive mode.
+    time_unit: microsecond
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2279
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/xxx
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-11-01"
+
+windows:
+  duration:
+    type: timing_distribution
+    description: >
+      Measuring how long windows keep opening.
```

```suggestion
      Measuring how long windows are kept open.
```

Was this what you meant?